python/HeaderService/hslib.py

```diff
@@ -21,6 +21,7 @@
 # Create a logger for all functions
 LOGGER = hutils.create_logger(level=logging.NOTSET, name='HEADERSERVICE')
 
+
```

Comment:
Whitespace/flake8 fixes should be in separate commits from functionality changes.

Reply:
Will separate these commits.